pyt/vulnerability_helper.py

```diff
@@ -92,88 +74,63 @@ def __str__(self):
             )
         )
 
+    def as_dict(self):
```

Comment:
We couldn't use `.__dict__` or `vars` and had to make our own `as_dict` function [like bandit](https://github.com/openstack/bandit/blob/3be3ca4de0b379d0d6713b28eec6c3b2ec9fcca9/bandit/core/issue.py#L107) because we had to call `.as_dict` on Node objects so that we didn't include all the Node attributes.